roles/forklift_versions/molecule/default/verify.yml

```diff
@@ -38,6 +38,22 @@
           - forklift_upgrade_version_intermediate == '3.16'
           - forklift_upgrade_version_final == 'nightly'
 
+    - name: "Include forklift_versions for upgrades with step=2"
+      include_role:
+        name: "forklift_versions"
+      vars:
+        scenario: "{{ pipeline_type }}"
+        scenario_os: "{{ pipeline_os }}"
+        scenario_version: "3.16"
+        upgrade: True
+        scenario_step: 2
```

Comment:
This doesn't match the `step` used everywhere.

Reply:
the playbook in the role uses `scenario_step` that gets passed to the `step` param, but yeah, alligning those is nice